L/LibGit2/build_tarballs.jl

```diff
@@ -15,10 +15,13 @@ cd $WORKSPACE/srcdir/libgit2*
 
 BUILD_FLAGS=(
     -DCMAKE_BUILD_TYPE=Release
+    -DBUILD_SHARED_LIBS=ON
     -DUSE_THREADS=ON
-    -DUSE_BUNDLED_ZLIB=ON
     -DUSE_SSH=ON
     -DBUILD_CLI=OFF
+    -DBUILD_TESTS=OFF
+    -DBUILD_EXAMPLES=OFF
+    -DBUILD_FUZZERS=OFF
```

Comment:
I disabled these (examples and fuzzers are off by default anyway, but better not to rely on defaults which can be changed in the future) because they only create noise in the log: tests are 2/3 of the entire build.